### PR TITLE
Add a note about `nix-shell` having to be run in plutus directory

### PR DESCRIPTION
**Pull request by arturaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@
   - Clone the [The Plutus repository](https://github.com/input-output-hk/plutus), check out the correct commit
     as specified in [cabal.project](code/week01/cabal.project).
   - Set-up IOHK binary caches [How to set up the IOHK binary caches](https://github.com/input-output-hk/plutus#iohk-binary-cache). "If you do not do this, you will end up building GHC, which takes several hours. If you find yourself building GHC, STOP and fix the cache."
-  - Enter a `nix-shell`.
+  - Enter a `nix-shell` in the Plutus repository directory with the correct commit checked out.
   - Build the [English Auction](code/week01) contract with `cabal build` (you may need to run `cabal update` first).
   - Go to the `plutus-playground-client` folder.
-  - Start the Playground server with `plutus-playground-server`.
-  - Start the Playground client (in another `nix-shell`) with `npm run start`.
+  - Start the Playground server with `plutus-playground-server`. When you will run this for the first time, a `nix` will build some things in the background, this will take time, be patient.
```
Comment (jhamPac) [inline README.md:29]: I think we can drop the word `will` in the first sentence. The following example below reads a little better.

"When you run this for the first time..."

File: README.md [hunk 2]
```diff
@@ -23,11 +23,11 @@
...
+  - Start the Playground client (in another `nix-shell`) with `npm run start`. When you will run this for the first time, a `nix` will build some things in the background, this will take time, be patient.
```
Comment (jhamPac) [inline README.md:30]: Same here,

"When you run this for the first time..."

**Review by jhamPac — CHANGES_REQUESTED** [review]
Thoughts?

File: README.md (hunk 1 above)
Comment (arturaz) [reply README.md]: Makes sense, updated.

**Comment by arturaz** [comment]
Makes sense, updated.

**Comment by luisantonioig** [comment]
Hi @arturaz,

I hope this message finds you well.

I wanted to inform you that we have decided to close this pull request as it pertains to an old branch of the course. Please note that these branches are not actively maintained and are kept for historical purposes.

We encourage you to utilize the branch that best suits your needs, whether it's the current version or any other relevant branch.

Please don't hesitate to reach out if you have any questions or require further assistance. We're here to help!

Thank you for your understanding and your valuable contributions.

**Pull request closed without merging**